libs/httpproxy/proxy-camel-servlet/src/main/java/org/codice/proxy/http/HttpProxyCamelHttpTransportServlet.java

```diff
@@ -280,7 +281,9 @@ protected HttpConsumer resolve(HttpServletRequest request) {
     HttpConsumer answer = consumers.get(endpointName);
 
     if (answer == null) {
-      log.debug("Consumer Keys: {}", Arrays.toString(consumers.keySet().toArray()));
+      if (LOGGER.isDebugEnabled()) {
+        log.debug("Consumer Keys: {}", Arrays.toString(consumers.keySet().toArray()));
```

Comment:
Is `Arrays.toString(consumers.keySet().toArray()));` necessary?  I think `consumers.keySet().toString()` would print a comma separated list of all the keys.

Reply:
You can even omit the call to `.toString()`